images/dockerregistry/config.yml

```diff
@@ -35,3 +35,8 @@ middleware:
         blobrepositorycachettl: 10m
   storage:
     - name: openshift
+openshift:
+  version: 1.0
+  metrics:
+    enabled: false
+    secret: TopSecretToken
```

Comment:
@legionus the problem is the fact that 'secret: TopSecretToken' is hardcoded and looks like nobody will change it by default, so we will end up with tons of clusters with this secret :-)